### PR TITLE
Makes debug verbs toggleable

### PR DESCRIPTION
**Pull request by lzimann**
It's kinda annoying when you click the debug verbs(the one that activates mapping buttons and such). So I made this.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
>clicking buttons instead of typing the commands

**Comment by lzimann** [comment]
They don't work if don't toggle them first.

File: code/modules/admin/verbs/mapping.dm [hunk 1]
```diff
@@ -20,6 +20,31 @@
 //- Check if the area has too much empty space. If so, make it smaller and replace the rest with maintenance tunnels.
 var/intercom_range_display_status = 0
 
+var/list/admin_verbs_debug_mapping = list(
+	/client/proc/do_not_use_these, 			//-errorage
```
Comment (KazeEspada) [inline code/modules/admin/verbs/mapping.dm]: 10/10 proc name

Comment (Jordie0608) [inline code/modules/admin/verbs/mapping.dm]: It makes sense in context

**Comment by MrStonedOne** [comment]
Don't add more verbs, add a var to the admin datum so you can make it be a proper toggle

File: code/modules/admin/verbs/mapping.dm [hunk 2]
```diff
@@ -20,6 +20,31 @@
...
+	/client/proc/sec_camera_report, 		//-errorage
+	/client/proc/intercom_view, 			//-errorage
+	/client/proc/air_status, //Air things
+	/client/proc/Cell, //More air things
+	/client/proc/atmosscan, //check plumbing
+	/client/proc/powerdebug, //check power
+	/client/proc/count_objects_on_z_level,
+	/client/proc/count_objects_all,
+	/client/proc/cmd_assume_direct_control,	//-errorage
+	/client/proc/startSinglo,
```
Comment (Iamgoofball) [inline code/modules/admin/verbs/mapping.dm]: i dont think this shit works anymore

File: code/modules/admin/verbs/mapping.dm [hunk 3]
```diff
@@ -20,6 +20,31 @@
...
+	/client/proc/fps,	//allows you to set the ticklag.
+	/client/proc/cmd_admin_grantfullaccess,
+	/client/proc/cmd_admin_areatest,
+	/client/proc/cmd_admin_rejuvenate,
```
Comment (Iamgoofball) [inline code/modules/admin/verbs/mapping.dm]: can we move this and GrantFullAccess to +ADMIN or something

File: code/modules/admin/verbs/mapping.dm (hunk 2 above)
Comment (MrStonedOne) [inline code/modules/admin/verbs/mapping.dm]: well maybe you should fix that, it worked fine before tesla =P

Comment (tkdrg) [inline code/modules/admin/verbs/mapping.dm]: I used this after tesla and it worked fine. Something else must have broken it.

**Pull request merged**